java-sdk/radar-schemas-tools/src/main/java/org/radarcns/schema/validation/rules/RadarSchemaRules.java

```diff
@@ -201,6 +202,27 @@ public SchemaFieldRules getFieldRules() {
                 messageSchema("\"" + TIME_RECEIVED + "\" is allow only in PASSIVE schemas."));
     }
 
+    @Override
+    public Validator<Schema> validateAvroData() {
+        return schema -> {
+            AvroDataConfig avroConfig = new AvroDataConfig.Builder()
+                    .with("connect.meta.data", false)
+                    .with("schemas.cache.config", 10)
+                    .with("enhanced.avro.schema.support", true)
```

Comment:
We can probably use constants from the AvroDataConfig class like `AvroDataConfig.CONNECT_META_DATA_CONFIG`.